public/components/panel/workstationPanel/constants.js

```diff
@@ -1,6 +1,7 @@
 const GRID_SCREEN_Y_OFFSET = 0
 const DEFAULT_GRID_COLUMN_COUNT = 16
 const MAXIMUM_GRID_COLUMN_COUNT = 100
+const PIXELS_FOR_SECOND = 2
```

Comment:
does this mean 1 pixel = 2 seconds? or for every 2 pixels thats 1 second?

Reply:
The latter